Week3/akif-week3.js

```diff
@@ -0,0 +1,158 @@
+//   STRINGS! --------------
+let myString = 'hello,this,is,a,difficult,to,read,sentence';
+console.log(myString);
+console.log(myString.length);
+myString = myString.replace(/[,]/g, ' ');
+console.log(myString);
+
+//   ARRAYS! -------------------------------------------------
+let favoriteAnimals = ['blowfish', 'capricorn', 'giraffe'];
+favoriteAnimals.push('turtle');
+console.log(favoriteAnimals);
+favoriteAnimals.splice(1, 0, 'meerkat');
+console.log("i added a new value 'meerkat' to the array");
+console.log(favoriteAnimals);
+console.log('The array has a length of: ' + favoriteAnimals.length);
+favoriteAnimals.splice(3, 1);
+console.log(favoriteAnimals);
+console.log('The item you are looking for is at index: ' + favoriteAnimals.indexOf('meerkat'));
+favoriteAnimals.splice(favoriteAnimals.indexOf('meerkat'), 1);
+
+//  MORE JAVASCRIPT ! ------------------------------------------------
+// -----1111111111111111111-----------
+function getSum(a, b, c) {
+  return a + b + c;
+}
+
+// -----222222222222222222-------------
+function colorCar(color) {
+  return 'a ' + color + ' car';
+}
+
+//-----3333333333333333333--------------
+const myBooks = {
+  Dostoyevsky: 'Crime and Punishment',
+  'Franz Kafka': 'The Trial',
+  'Anne Frank': 'Diarys',
+};
+
+function getBook(library) {
+  for (let book in library) {
+    console.log(book + ' - ' + library[book]);
+  }
+}
+getBook(myBooks);
+
+//  -----444444444444444444------------------
+const vehicleCodes = {
+  1: 'car',
+  2: 'motorbike',
+};
+
+function vehicleType(color, code) {
+  console.log('A ' + color + ' ' + vehicleCodes[code]);
+}
+
+vehicleType('blue', 2);
+
+//  _-------555555555555555555--------------------
+console.log(3 === 3 ? 'yes' : 'no');
+
+//  -------66666666666666666666---------------------
+function vehicle(color, code, age) {
+  if (age > 0) {
+    age = 'used';
+  } else {
+    age = 'new';
+  }
```

Comment:
Umm... I would maybe not overwrite `age` with a new value and instead do something like `const type = (age > 0) ? 'used' : 'new';` 🙂